kommunicate/src/main/java/io/kommunicate/Kommunicate.java

```diff
@@ -882,4 +884,53 @@ public static void loadAwayMessage(Context context, Integer groupId, KmAwayMessa
     public static void removeApplicationKey(Context context) {
         new SecureSharedPreferences(AlPrefSettings.AL_PREF_SETTING_KEY, ApplozicService.getContext(context)).edit().remove("APPLICATION_KEY").commit();
     }
+
+    public static void openZendeskChat(final Context context) {
```

Comment:
is it possible to move all Zendesk related code to separate class files and from here keep the minimum few lines to call those methods.
This way it will be easy for us to maintain/merge/update the repos when new Kommunicate SDK code changes get merged to Zendesk code base branch